src/nmea.cpp

```diff
@@ -945,6 +957,14 @@ void GPSDriverNMEA::handleHeading(float heading_deg, float heading_stddev_deg)
 	_gps_position->heading_accuracy = heading_stddev_rad;
 }
 
+void GPSDriverNMEA::request_unicore_heading_message()
+{
+	// Configure heading message on serial port at 5 Hz. Don't save it though.
+	uint8_t buf[] = "UNIHEADINGA COM1 0.2\r\n";
+	write(buf, sizeof(buf));
```

Comment:
The terminating null should not be written I assume.
```suggestion
	write(buf, sizeof(buf)-1);
```

Reply:
Right.